frontend/app/model/Eventbrite.scala

```diff
@@ -171,6 +185,9 @@ object Eventbrite {
     val imgUrl = data.flatMap(_.images.headOption).flatMap(_.file)
       .getOrElse(views.support.Asset.at("images/event-placeholder.gif"))
       .replace("http://static", "https://static-secure")
+
+    val availableWidths = ""
```

Comment:
We can get available widths for Masterclasses too as CAPI provides this. I'll do this as a separate PR as Masterclasses doesn't use the Grid.